code/game/objects/effects/spiderwebs.dm

```diff
@@ -132,3 +164,41 @@
 	for(var/atom/movable/A in contents)
 		A.forceMove(T)
 	return ..()
+
+/obj/structure/spider/sticky
+	name = "sticky web"
+	icon = 'icons/effects/effects.dmi'
+	desc = "Extremely soft and sticky silk."
+	icon_state = "verystickyweb"
+	max_integrity = 20
+
+/obj/structure/spider/sticky/CanAllowThrough(atom/movable/mover, border_dir)
+	. = ..()
+	if(isspider(mover))
+		return TRUE
+	if(!isliving(mover))
+		return
+	if(isspider(mover.pulledby))
+		return TRUE
+	balloon_alert(mover, "stuck in web!")
+	return FALSE
+
+/obj/structure/spider/spikes
+	name = "web spikes"
+	icon = 'icons/effects/effects.dmi'
+	desc = "hardened silk formed into small yet deadly spikes."
+	icon_state = "webspikes1"
+	max_integrity = 40
+
+/obj/structure/spider/spikes/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/caltrop, min_damage = 20, max_damage = 30, flags = CALTROP_NOSTUN | CALTROP_BYPASS_SHOES)
```

Comment:
this is way way too much damage

Reply:
tested it for around 200 rounds it is not

Reply:
each limb takes up to only 50 damage and if each leg has 50 each it doesn't crit

Reply:
making it any less would make the spikes trivial

Reply:
> this is way way too much damage

Also is the exact same damage as punji bamboo spikes BUT it doesn't insta perma stun which essentially makes this a weaker version of something crewmembers can make with a 40 second cooldown

Reply:
![image](https://github.com/tgstation/tgstation/assets/84478872/6f86c4b8-6ee8-4067-8d8d-b39885761292)